### PR TITLE
Update feature-release.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated the working days description based on: https://github.com/mattermost/mattermost-developer-documentation/pull/442 for consistency.

### CODE REVIEW

**Comment by justinegeffen** [comment]
Updated phrasing around working days in line with https://github.com/mattermost/mattermost-developer-documentation/pull/442 but retained working days as opposed to business days for consistency across the doc. 

Happy to change working days to business days if no one is opposed to it.

**Comment by amyblais** [comment]
I'm 0/5. I follow https://docs.mattermost.com/process/working-at-mattermost.html#holidays when I count the T-x days.

Maybe we can update it to say ``excluding [the listed statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays)`` or similar.

**Comment by jasonblais** [comment]
Good to know! Linking to that list may indeed help clarify the point

File: source/process/feature-release.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Recommended pre-reading:
 
 Notes:
 - All cut-off dates are based on 10am ([San Francisco Time](http://everytimezone.com/)) on the day stated.
-- T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
+- T-minus counts are measured in "working days" (weekdays, Monday through Friday, excluding [statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays)) prior to release day.
```
Comment (amyblais) [inline source/process/feature-release.md]:
```suggestion
- T-minus counts are measured in "working days" (weekdays, Monday through Friday, excluding [the listed statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays)) prior to release day.
```

Comment (justinegeffen) [reply source/process/feature-release.md]: Thanks for the edit - that makes sense!

**Pull request merged**